coreneuron/nrniv/cellorder.cpp

```diff
@@ -65,13 +65,13 @@ InterleaveInfo& InterleaveInfo::operator=(const InterleaveInfo& info) {
 
 InterleaveInfo::~InterleaveInfo() {
```

Comment:
The `InterleaveInfo` class is a good candidate for `std::unique_ptr` instead of manual new/delete.